### ISSUE TITLE
[complete]Please check for reinstallation errors(Host is not registered)

### ISSUE THREAD

**Issue by BLUEDIAA** [EvernodeXRPL/evernode-host#332]
#1  When reinstalling, the error below occurs:
Specify the Xahau account address: rnhm4tMeYadubJNvwoB757rqK5yb9EbD1H
Checking account rnhm4tMeYadubJNvwoB757rqK5yb9EbD1H...
The account needs minimum balance of 500 EVR. Current balance is 3 EVR.
Specify the Xahau account address: 

#2 When installing with transfer settings
Specify the Xahau account address: rnhm4tMeYadubJNvwoB757rqK5yb9EbD1H
Checking account rnhm4tMeYadubJNvwoB757rqK5yb9EbD1H...
Host is not registered.
Specify the Xahau account address: 

# In conclusion, I have already transferred 500 evr and 17.6 xah, 
and an error occurred during the installation completion stage. 
So, I try to reinstall as above, but errors occur on both sides.

Administrator, please help me solve this problem. 
thank you

**Comment by BLUEDIAA** [comment]
The case is below
https://xahauexplorer.com/nft/nft Code URIToken has been successfully completed through repurchase approval.
thank you
